### PR TITLE
Add ERC: Atomic Push-based Data Feed Among Contracts

### PR DESCRIPTION
**Pull request by lanyinzly**
Add ERC: Smart Contract Data Push Mechanism

An Atomic Mechanism to Allow Publisher Contract Push Data to Subscriber Contracts

### CODE REVIEW

File: ERCS/ERC-xxxx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+eip: xxxx
```
Comment (abcoathup) [inline ERCS/ERC-xxxx:2]: To be assigned an ERC number a PR needs a specification as a minimum.

Comment (lanyinzly) [reply ERCS/ERC-xxxx:2]: updated

File: ERCS/ERC-xxxx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
```
Comment (abcoathup) [inline ERCS/ERC-xxxx:2]:
```suggestion
eip: 7615
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

**Comment by lanyinzly** [comment]
It says "All Reviewers Have Approved; Performing Automatic Merge..." on Feb 18, but it still requires review and cannot be merged. Any feedback or could you help to merge? @axic , @g11tech , @SamWilsn , @xinbenlv

File: ERCS/erc-7615.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+### Using Function Selectors as Parameters
+
+Using function selectors to retrieve the addresses of subscriber contracts allows 
+more detailed configuration. 
+For the subscriber contract, having the specific function of the request source based on the push information enables more accurate handling of the push information.
+
+## Reference Implementation
+
+```solidity
+// SPDX-License-Identifier: MIT
```
Comment (SamWilsn) [inline ERCS/erc-7615.md:157]: Code in the document itself must be licensed under `CC0-1.0`. I allow MIT licensed code in the `assets/` directory.

**Review by SamWilsn — APPROVED** [review]

**Review by SamWilsn — DISMISSED** [review]

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**